src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/techniques/normal/ScaffoldStabilizeMovementFeature.kt

```diff
@@ -29,7 +29,8 @@ import net.ccbluex.liquidbounce.utils.movement.getDegreesRelativeToView
 import net.ccbluex.liquidbounce.utils.movement.getDirectionalInputForDegrees
 import net.minecraft.util.math.Vec3d
 
-object ScaffoldStabilizeMovementFeature : ToggleableConfigurable(ScaffoldNormalTechnique, "StabilizeMovement", true) {
+object ScaffoldStabilizeMovementFeature : ToggleableConfigurable(ScaffoldNormalTechnique, "StabilizeMovement",
+    false) {
```

Comment:
Ayo, why is this off by default?

Reply:
because stabilize movement is almost never useful

Reply:
> Ayo, why is this off by default?

Stabilise Movement can screw people up, it’s easier to control without it and it’ll feel better to people that have used other blatant clients before - they won’t be shoved around.

Reply:
Stabilize Movement does not shove the user around, it forces the player on a line. I think it is more problematic when people try to use the scaffold, but it flags because the player is moving on a line which makes it impossible for the client to find good rotations for.